pricing-page-exercise/salih/styles.css

```diff
@@ -0,0 +1,195 @@
+/* Global */
+
+*, ::after, ::before {
+    box-sizing: border-box;
+    font-family:sans-serif;
```

Comment:
It's more common to add font-family to either `html` or `body` selector instead.

Reply:
I moved to `html`.